From 96c9a3279cf5b3a35493b6761b866c3bbbc72759 Mon Sep 17 00:00:00 2001
From: Pavan Naregundi <pnaregundi@marvell.com>
Date: Thu, 12 Oct 2023 09:32:17 +0530
Subject: arm64: Select CONFIG_SPI_ORION

This patch selects CONFIG_SPI_ORION with 'm' to fix mtd partition
detection issue.

Change-Id: I3a59adfa7d9a03822cf555ea32c78954ef7e51af
Signed-off-by: Pavan Naregundi <pnaregundi@marvell.com>
---
 patch/kconfig-inclusions | 1 +
 1 file changed, 1 insertion(+)

diff --git a/patch/kconfig-inclusions b/patch/kconfig-inclusions
index 4686349..fdf5349 100644
--- a/patch/kconfig-inclusions
+++ b/patch/kconfig-inclusions
@@ -108,6 +108,7 @@ CONFIG_MTD_NAND_MARVELL=m
 CONFIG_SENSORS_EMC2305=m
 CONFIG_KEXEC_FILE=y
 CONFIG_PHY_MVEBU_CP110_COMPHY=y
+CONFIG_SPI_ORION=m
 
 [marvell-arm64]
 CONFIG_MARVELL_AC5_WATCHDOG=m
-- 
2.25.1

